docs/INSTALL.md

```diff
@@ -96,17 +96,21 @@ fprime-util install --jobs 32
 fprime-gds -g html -d <path to fprime checkout>/Ref
 ```
 **Note:** `Ref` should contain pre-built dictionaries and binaries for the user's system. This can
-be achieved by running the autocoder installation test (the user must have a working autocoder
+be achieved by running the Autocoder installation test (the user must have a working Autocoder
 installation).
 
+## (Optional) Installing Tab Completion
+
+Several of F´s command line utilities support tab completion. To enable these tools to use it, see the [instructions here](../Gds/docs/gdsCLI/AUTOCOMPLETE.md).
```

Comment:
You should put this document in the normal `docs` folder.

`fprime/docs/gds/AUTOCOMPLETE.md`

Reply:
Can do!

Reply:
Should we move the Test API documentation too, or is that different because it isn't needed for the installation process?